ambari-server/src/test/java/org/apache/ambari/server/upgrade/UpgradeCatalog270Test.java

```diff
@@ -952,13 +953,13 @@ public void shouldNotSaveLdapConfigurationIfPropertyIsNotSetInAmbariProperties()
-
   @Test
   public void testupdateKerberosDescriptorArtifact() throws Exception {
+    String kerberosDescriptorJson = FileUtils.readFileToString(new File("src/test/resources/org/apache/ambari/server/upgrade/kerberos_descriptor.json"), "UTF-8");
```

Comment:
is it possible to read from the classpath? not sure that can work from everywhere (based on where is the working directory e.g. like in ambari, ambari-server etc.)